src/Microsoft.AspNetCore.Server.IISIntegration/NativeMethods.cs

```diff
@@ -7,22 +7,14 @@
 
 namespace Microsoft.AspNetCore.Server.IISIntegration
 {
-    internal class NativeMethods
+    internal static class NativeMethods
     {
-#if DOTNET5_4
```

Comment:
@Tratcher this was here for out of process. Is it safe to remove now?

Reply:
yes, we never built that. It was for nano (?)